chain/rust/src/builders/certificate_builder.rs

```diff
@@ -51,17 +52,23 @@ pub fn cert_required_wits(cert: &Certificate, required_witnesses: &mut RequiredW
         Certificate::PoolRetirement(cert) => {
             required_witnesses.add_vkey_key_hash(cert.ed25519_key_hash.clone());
         }
-        Certificate::GenesisKeyDelegation(cert) => {
-            required_witnesses.add_vkey_key_hash(
-                Ed25519KeyHash::from_raw_bytes(cert.genesis_delegate_hash.to_raw_bytes()).unwrap(),
-            );
-        }
-        // no witness as there is no single core node or genesis key that posts the certificate
-        Certificate::MoveInstantaneousRewardsCert(_cert) => {}
+        Certificate::RegCert(_cert) => todo!(),
```

Comment:
will we introduce that in the following prs?

Reply:
Yes